2024.7.0

frenck opened the pull request:
### ⚠️ DO NOT MERGE

This PR is here during the beta week, so we can keep an eye on our CI.

../Frenck

Lash-L committed:
Bump python-roborock to 2.5.0 (#120466)

teharris1 committed:
Bump pyinsteon to 1.6.3 to fix Insteon device status (#120464)

thomaskistler committed:
Add @thomaskistler as an owner of hydrawise (#120477)

bdraco committed:
Bump uiprotect to 3.7.0 (#120471)

cdce8p committed:
Improve vol.Invalid handling (#120480)

cdce8p committed:
Improve SERVICE_TO_METHOD typing (#120474)

EnjoyingM committed:
Bump wolf-comm to 0.0.9 (#120473)

cdce8p committed:
Improve schema typing (2) (#120475)

cdce8p committed:
Improve config vol.Invalid typing (#120482)

bdraco committed:
Fix dropped unifiprotect motion events (#120489)

Commit:
Align deviceinfo entries in pyLoad integration (#120478)

gjohansson-ST committed:
Fix holiday using utc instead of local time (#120432)

Commit:
Add missing textselectors in `USER_DATA_SCHEMA` in pyLoad integration (#120479)

Commit:
Remove unused translation strings in pyLoad integration (#120476)

Grubalex committed:
Add Philips WiZ Lightbulbs to Matter TRANSITION_BLOCKLIST (#120461)

bdraco committed:
Make fetching integrations with requirements safer (#120481)

jbouwh committed:
Cleanup mqtt platform tests part 2 (#120490)

emontnemery committed:
Deduplicate alarm_control_panel services.yaml (#118796)

edenhaus committed:
Create repair when HA auth provider is running in legacy mode (#119975)

abmantis committed:
Handle availability in Idasen Desk height sensor (#120277)

synesthesiam committed:
Allow timer management from any device (#120440)

jbouwh committed:
Fix issue where an MQTT device is removed linked to two config entries (#120430)

* Fix issue where an MQTT device is removed linked to two config entries

* Update homeassistant/components/mqtt/discovery.py

Co-authored-by: J. Nick Koston <nick@koston.org>

* Update homeassistant/components/mqtt/debug_info.py

Co-authored-by: J. Nick Koston <nick@koston.org>

---------

Co-authored-by: J. Nick Koston <nick@koston.org>

bdraco committed:
Bump dbus-fast to 2.22.1 (#120491)

thomaskistler committed:
Add active watering time sensor to Hydrawise (#120177)

joostlek committed:
Add unique_id to MPD (#120495)

jbouwh committed:
Cleanup mqtt platform tests part 3 (#120493)

starkillerOG committed:
Do not wait for Reolink firmware check (#120377)

gjohansson-ST committed:
Add timezone testing in holiday (#120497)

emontnemery committed:
Verify default timezone is restored when test ends (#116216)

Commit:
Add reconfiguration flow to pyLoad integration (#120485)

joostlek committed:
Remove Knocki triggers on runtime (#120452)

* Bump Knocki to 0.2.0

* Remove triggers on runtime in Knocki

* Fix

epenet committed:
Force alias when importing notify PLATFORM_SCHEMA (#120494)

epenet committed:
Force alias when importing air_quality PLATFORM_SCHEMA (#120502)

Commit:
Add switch platform to pyload integration (#120352)

epenet committed:
Force alias when importing tts PLATFORM_SCHEMA (#120500)

bouwew committed:
Implement @plugwise_command for Plugwise Number platform (#120503)

arturpragacz committed:
Switch onkyo to pyeiscp, making it local_push (#120026)

* Switch onkyo to pyeiscp, making it local_push

Major rewrite of the integration to use pyeiscp.
This facilitates use of the async push updates.
Streamline the code dealing with zones.
Handle sound mode.
Add myself to codeowners.

* Add types

* Add more types

* Address feedback

* Remove sound mode support for now

* Fix zone detection

* Keep legacy unique_id

epenet committed:
Force alias when importing switch PLATFORM_SCHEMA (#120504)

epenet committed:
Force alias when importing calendar PLATFORM_SCHEMA (#120512)

epenet committed:
Force alias when importing alarm control panel PLATFORM_SCHEMA (#120505)

Commit:
Rename sensor to finished downloads in pyLoad integration (#120483)

epenet committed:
Force alias when importing binary sensor PLATFORM_SCHEMA (#120510)

epenet committed:
Force alias when importing climate PLATFORM_SCHEMA (#120518)

Commit:
Add exception translations for pyLoad integration (#120520)

epenet committed:
Force alias when importing camera PLATFORM_SCHEMA (#120514)

bouwew committed:
Implement @plugwise_command for Plugwise Select platform (#120509)

epenet committed:
Force alias when importing device tracker PLATFORM_SCHEMA (#120523)

emontnemery committed:
Simplify EVENT_STATE_REPORTED (#120508)

epenet committed:
Force alias when importing humidifier PLATFORM_SCHEMA (#120526)

epenet committed:
Force alias when importing geo location PLATFORM_SCHEMA (#120525)

epenet committed:
Force alias when importing cover PLATFORM_SCHEMA (#120522)

epenet committed:
Force alias when importing image processing PLATFORM_SCHEMA (#120527)

cdce8p committed:
Improve schema typing (3) (#120521)

hahn-th committed:
Add HmIP-ESI (#116863)

epenet committed:
Force alias when importing event and fan PLATFORM_SCHEMA (#120524)

agners committed:
Improve Matter Server version incompatibility handling (#120416)

* Improve Matter Server version incompatibility handling

Improve the handling of Matter Server version. Noteably fix the issues
raised (add strings for the issue) and split the version check into
two cases: One if the server is too old and one if the server is too
new.

* Bump Python Matter Server library to 6.2.0b1

* Address review feedback

Commit:
Add diagnostics platform to pyLoad integration (#120535)

epenet committed:
Force alias when importing remote PLATFORM_SCHEMA (#120533)

Bre77 committed:
Refactor Tessie for future PR (#120406)

* Bump tessie-api

* Refactor

* revert bump

* Fix cover

* Apply suggestions from code review

Co-authored-by: G Johansson <goran.johansson@shiftit.se>

---------

Co-authored-by: G Johansson <goran.johansson@shiftit.se>

epenet committed:
Force alias when importing lock PLATFORM_SCHEMA (#120531)

epenet committed:
Force alias when importing light PLATFORM_SCHEMA (#120529)

epenet committed:
Force alias when importing scene PLATFORM_SCHEMA (#120534)

epenet committed:
Force alias when importing sensor PLATFORM_SCHEMA (#120536)

bouwew committed:
Import DOMAIN constants for Plugwise and implement (#120530)

joostlek committed:
Add learning offset select to Airgradient (#120532)

epenet committed:
Force alias when importing media player PLATFORM_SCHEMA (#120537)

emontnemery committed:
Correct type annotations in integration sensor tests (#120541)

emontnemery committed:
Add primary_config_entry attribute to device registry entries (#119959)

Co-authored-by: Franck Nijhof <git@frenck.dev>
Co-authored-by: Robert Resch <robert@resch.dev>

rikroe committed:
Improve BMW tests (#119171)

Co-authored-by: Richard <rikroe@users.noreply.github.com>

epenet committed:
Revert "Force alias when importing scene PLATFORM_SCHEMA" (#120540)

Revert "Force alias when importing scene PLATFORM_SCHEMA (#120534)"

This reverts commit 348ceca19f1fe5b45dadbbd7ec96093c64409a3f.

joostlek committed:
Fix release in MPD issue (#120545)

Commit:
Remove unnecessary icon states in pyLoad integration (#120548)

Remove unnecessary icon states

edenhaus committed:
Fix airgradient select entities (#120549)

bramkragten committed:
Update frontend to 20240626.0 (#120546)

emontnemery committed:
Use state_reported events in Riemann sum sensor (#113869)

Commit:
Convert dataclass to dict in pyLoad diagnostics (#120552)

Commit:
Fix class and variable naming errors in pyLoad integration (#120547)

bouwew committed:
Implement remaining select-adaptions for Plugwise (#120544)

Commit:
Update docstrings in pyLoad tests (#120556)

joostlek committed:
Increase max temperature to 40 for Tado (#120560)

Lash-L committed:
Add Roborock to strict typing (#120379)

Commit:
Add measurement unit and state_class to sensors in pyLoad (#120551)

synesthesiam committed:
Add created_seconds to timer info and pass to ESPHome devices (#120364)

joostlek committed:
Add switch platform to Airgradient (#120559)

epenet committed:
Use ruff to force alias when importing PLATFORM_SCHEMA (#120539)

Bre77 committed:
Add new coordinators to Tessie (#118452)

* WIP

* wip

* Add energy classes

* Add basis for Testing

* Bump Library

* fix case

* bump library

* bump library again

* bump library for teslemetry

* reorder

* Fix super

* Update strings.json

* Tests

* Small tweaks

* Bump

* Bump teslemetry

* Remove version

* Add WC states

* Bump to match dev

* Review feedback

Co-authored-by: Joost Lekkerkerker <joostlek@outlook.com>

* Review feedback

* Review feedback 1

* Review feedback 2

* TessieWallConnectorStates Enum

* fixes

* Fix translations and value

* Update homeassistant/components/tessie/strings.json

* Update homeassistant/components/tessie/strings.json

---------

Co-authored-by: Joost Lekkerkerker <joostlek@outlook.com>

Lash-L committed:
Add options flow to Roborock (#104345)

Co-authored-by: Robert Resch <robert@resch.dev>

rytilahti committed:
Adjust tplink codeowners (#120561)

Commit:
Add Base class for entities in PyLoad integration (#120563)

* Add Base class for entities

* Remove constructors

Commit:
Redact the hostname in pyLoad diagnostics (#120567)

Commit:
Add value_fn to switch entity description in pyLoad (#120569)

bouwew committed:
Implement a reboot-button for Plugwise (#120554)

Co-authored-by: Franck Nijhof <frenck@frenck.nl>

Lash-L committed:
Add support for Dyad vacuums to Roborock (#115331)

joostlek committed:
Disable Aladdin Connect (#120558)

Co-authored-by: Robert Resch <robert@resch.dev>

Commit:
Fix class name and deprecation version (#120570)

epenet committed:
Import PLATFORM_SCHEMA from platform not from helpers (#120565)

epenet committed:
Refactor PLATFORM_SCHEMA imports in platforms (#120564)

emontnemery committed:
Add explanatory comment in tests/patch_time.py (#120572)

treetip committed:
Add profile duration sensor for Vallox integration (#120240)

gjohansson-ST committed:
Add config flow to statistics (#120496)

Commit:
Add platinum scale to pyLoad integration (#120542)

Add platinum scale

bouwew committed:
Delete removed device(s) at runtime in Plugwise (#120296)

epenet committed:
Refactor PLATFORM_SCHEMA imports in tests (#120566)

gjong committed:
Add additional tests to youless integration (#118869)

frenck committed:
Bump version to 2024.7.0b0

frenck commented:
@rikroe The BMW tests seem to be flaky, could you take a look? See: <https://github.com/home-assistant/core/actions/runs/9681391957/job/26712283615?pr=120579>

synesthesiam committed:
Bump intents to 2024.6.26 (#120584)

Bump intents

TheJulianJES committed:
Bump ZHA dependencies (#120581)

thecode committed:
Change Shelly connect task log message level to error (#120582)

rikroe commented:
> @rikroe The BMW tests seem to be flaky, could you take a look? See: https://github.com/home-assistant/core/actions/runs/9681391957/job/26712283615?pr=120579

Probably related to #119171. Unfortunately I will only be able to check this in more detail tomorrow, currently without a PC.

thecode committed:
Add last_error reporting to Shelly diagnostics (#120595)

thecode committed:
Align Shelly sleeping devices timeout with non-sleeping (#118969)

jbouwh commented:
> > @rikroe The BMW tests seem to be flaky, could you take a look? See: https://github.com/home-assistant/core/actions/runs/9681391957/job/26712283615?pr=120579
> 
> Probably related to #119171. Unfortunately I will only be able to check this in more detail tomorrow, currently without a PC.

May be this helps: https://github.com/home-assistant/core/pull/120600

elcajon committed:
Correct deprecation warning `async_register_static_paths` (#120592)

emontnemery committed:
Remove deprecated run_immediately flag from integration sensor (#120593)

jbouwh committed:
Prevent changes to mutable bmw_connected_drive fixture data (#120600)

cdce8p committed:
Fix mqtt test fixture usage (#120602)

florian-sabonchi committed:
Don't allow switch toggle when device in locked in AVM FRITZ!SmartHome (#120132)

* fix: set state of the FritzBox-Switch to disabled if the option for manuel switching in the userinterface is disabled

* feat: raise an error instead of disabling switch

* feat: rename method signature

* fix: tests

* fix: wrong import

* feat: Update homeassistant/components/fritzbox/strings.json

Update error message

Co-authored-by: Michael <35783820+mib1185@users.noreply.github.com>

* Update tests/components/fritzbox/test_switch.py

feat: update test

Co-authored-by: Michael <35783820+mib1185@users.noreply.github.com>

* make ruff happy

* fix expected error message check

---------

Co-authored-by: Michael <35783820+mib1185@users.noreply.github.com>

mj23000 committed:
Improve Bang & Olufsen error messages (#120587)

* Convert logger messages to raised errors where applicable

* Modify exception types

* Improve deezer / tidal error message

* Update homeassistant/components/bang_olufsen/strings.json

Co-authored-by: Michael <35783820+mib1185@users.noreply.github.com>

* Update homeassistant/components/bang_olufsen/media_player.py

Co-authored-by: Michael <35783820+mib1185@users.noreply.github.com>

---------

Co-authored-by: Michael <35783820+mib1185@users.noreply.github.com>

frenck committed:
Update adguardhome to 0.7.0 (#120605)

frenck committed:
Bump version to 2024.7.0b1

luca-angemi committed:
Fix telegram bot thread_id key error (#120613)

piitaya committed:
Update frontend to 20240626.2 (#120614)

frenck committed:
Bump version to 2024.7.0b2

jbouwh committed:
Cleanup mqtt platform tests part 4 (init) (#120574)

Lash-L committed:
Bump anova_wifi to 0.14.0 (#120616)

bdraco committed:
Remove unused fields from unifiprotect event sensors (#120568)

bdraco committed:
Bump uiprotect to 4.0.0 (#120617)

bdraco committed:
Add async_track_state_reported_event to fix integration performance regression (#120622)

split from https://github.com/home-assistant/core/pull/120621

bdraco committed:
Fix performance regression in integration from state_reported (#120621)

* Fix performance regression in integration from state_reported

Because the callbacks were no longer indexed by entity id, users
saw upwards of 1M calls/min

https://github.com/home-assistant/core/pull/113869/files#r1655580523

* Update homeassistant/helpers/event.py

* coverage

---------

Co-authored-by: Paulus Schoutsen <balloob@gmail.com>

zweckj committed:
Fix the version that raises the issue (#120638)

emontnemery committed:
Don't allow updating a device to have no connections or identifiers (#120603)

* Don't allow updating a device to have no connections or identifiers

* Move check to the top of the function

zweckj committed:
Move Auto On/off switches to Config EntityCategory (#120648)

emontnemery committed:
Add snapshots to tasmota sensor test (#120647)

joostlek committed:
Disable polling for Knocki (#120656)

joostlek committed:
Fix Airgradient ABC days name (#120659)

emontnemery committed:
Bump hatasmota to 0.9.1 (#120649)

emontnemery committed:
Improve typing of state event helpers (#120639)

jesserockz committed:
[esphome] Add more tests to bring integration to 100% coverage (#120661)

Bre77 committed:
Fix values at startup for Tessie (#120652)

jesserockz committed:
Mark esphome integration as platinum (#112565)

Co-authored-by: J. Nick Koston <nick@koston.org>

emontnemery committed:
Rename async_track_state_reported_event to async_track_state_report_event (#120637)

* Rename async_track_state_reported_event to async_track_state_report_event

* Update tests

sdb9696 committed:
Store tplink credentials_hash outside of device_config (#120597)

sdb9696 committed:
Update tplink unlink identifiers to deal with ids from other domains (#120596)

emontnemery committed:
Fix docstring for EventStateEventData (#120662)

gwww committed:
Bump Environment Canada to 0.7.0 (#120686)

bdraco committed:
Add newer models to unifi integrations discovery (#120688)

jbouwh committed:
Split mqtt client tests (#120636)

joostlek committed:
Fix unknown attribute in MPD (#120657)

gwww committed:
Bump upb-lib to 0.5.7 (#120689)

MatthewFlamm committed:
Use more observations in NWS (#120687)

Use more observations

bdraco committed:
Bump uiprotect to 4.2.0 (#120669)

Thomas55555 committed:
Bump aioautomower to 2024.6.3 (#120697)

ALERTua committed:
Improve AtlanticDomesticHotWaterProductionMBLComponent support in Overkiz (#114178)

* add overkiz AtlanticDHW support

Adds support of Overkiz water heater entity selection based on device controllable_name
Adds support of Atlantic water heater based on Atlantic Steatite Cube WI-FI VM 150 S4CS 2400W
Adds more Overkiz water heater binary_sensors, numbers, and sensors

* Changed class annotation

* min_temp and max_temp as properties

* reverted binary_sensors, number, sensor to make separate PRs

* Update homeassistant/components/overkiz/water_heater_entities/atlantic_dhw.py

Co-authored-by: Mick Vleeshouwer <mick@imick.nl>

* Update homeassistant/components/overkiz/water_heater_entities/atlantic_dhw.py

Co-authored-by: Mick Vleeshouwer <mick@imick.nl>

* Update homeassistant/components/overkiz/water_heater_entities/atlantic_dhw.py

Co-authored-by: Mick Vleeshouwer <mick@imick.nl>

* Update homeassistant/components/overkiz/water_heater.py

Co-authored-by: Mick Vleeshouwer <mick@imick.nl>

* Update homeassistant/components/overkiz/water_heater_entities/atlantic_dhw.py

Co-authored-by: Mick Vleeshouwer <mick@imick.nl>

* Update homeassistant/components/overkiz/water_heater_entities/atlantic_dhw.py

Co-authored-by: Mick Vleeshouwer <mick@imick.nl>

* review fixes, typos, and pylint

* review fix

* review fix

* ruff

* temperature properties changed to constructor attributes

* logger removed

* constants usage consistency

* redundant mapping removed

* Update homeassistant/components/overkiz/water_heater_entities/atlantic_dhw.py

Co-authored-by: Mick Vleeshouwer <mick@imick.nl>

* boost mode method annotation typo

* removed away mode for atlantic dwh

* absence and boost mode attributes now support 'prog' state

* heating status bugfix

* electrical consumption sensor

* warm water remaining volume sensor

* away mode reintroduced

* mypy check

* boost plus state support

* Update homeassistant/components/overkiz/sensor.py

Co-authored-by: Mick Vleeshouwer <mick@imick.nl>

* sensors reverted to separate them into their own PR

* check away and boost modes on before switching them off

* atlantic_dhw renamed to atlantic_domestic_hot_water_production

* annotation changed

* AtlanticDomesticHotWaterProductionMBLComponent file renamed, annotation change reverted

---------

Co-authored-by: Mick Vleeshouwer <mick@imick.nl>

bramkragten committed:
Update frontend to 20240627.0 (#120693)

joostlek committed:
Bump ttls to 1.8.3 (#120700)

gwww committed:
Bump Environment Canada to 0.7.1 (#120699)

bdraco committed:
Bump unifi-discovery to 1.2.0 (#120684)

frenck committed:
Bump version to 2024.7.0b3

emontnemery committed:
Bump hatasmota to 0.9.2 (#120670)

bieniu committed:
Bump `nextdns` to version 3.1.0 (#120703)

Co-authored-by: Maciej Bieniek <478555+bieniu@users.noreply.github.com>

joostlek committed:
Bump apsystems-ez1 to 1.3.3 (#120702)

emontnemery committed:
Set stateclass on unknown numeric Tasmota sensors (#120650)

danzel committed:
Bump airtouch5py to 1.2.0 (#120715)

* Bump airtouch5py to fix console 1.2.0

* Bump airtouch5py again

ALERTua committed:
Add electrical consumption sensor to Overkiz (#120717)

electrical consumption sensor

ALERTua committed:
Add warm water remaining volume sensor to Overkiz (#120718)

* warm water remaining volume sensor

* Update homeassistant/components/overkiz/sensor.py

Co-authored-by: Dave T <17680170+davet2001@users.noreply.github.com>

---------

Co-authored-by: Dave T <17680170+davet2001@users.noreply.github.com>

epenet committed:
Bump renault-api to 0.2.4 (#120727)

joostlek committed:
Bump panasonic_viera to 0.4.2 (#120692)

* Bump panasonic_viera to 0.4.2

* Bump panasonic_viera to 0.4.2

* Bump panasonic_viera to 0.4.2

* Fix Keys

Bre77 committed:
Check Tessie scopes to fix startup bug (#120710)

* Add scope check

* Add tests

* Bump Teslemetry

ikalnyi committed:
Bump asyncarve to 0.1.1 (#120740)

dougiteixeira committed:
Link the Statistics helper entity to the source entity device (#120705)

tronikos committed:
Fix Google Generative AI: 400 Request contains an invalid argument (#120741)

epenet committed:
Separate renault strings (#120737)

frenck committed:
Bump version to 2024.7.0b4

joostlek committed:
Bump sense-energy to 0.12.4 (#120744)

* Bump sense-energy to 0.12.4

* Fix

klaasnicolaas committed:
Bump easyenergy lib to v2.1.2 (#120753)

klaasnicolaas committed:
Bump p1monitor lib to v3.0.1 (#120756)

joostlek committed:
Bump govee-local-api to 1.5.1 (#120747)

bdraco committed:
Increase mqtt availablity timeout to 50s (#120760)

joostlek committed:
Bump aiowithings to 3.0.2 (#120765)

bieniu committed:
Do not call async_delete_issue() if there is no issue to delete in Shelly integration (#120762)

Co-authored-by: Maciej Bieniek <478555+bieniu@users.noreply.github.com>

chamberlain2007 committed:
Bump pyaprilaire to 0.7.4 (#120782)

klaasnicolaas committed:
Bump energyzero lib to v2.1.1 (#120783)

wittypluck committed:
Reject small uptime updates for Unifi clients (#120398)

Extend logic to reject small uptime updates to Unifi clients + add unit tests

klaasnicolaas committed:
Bump odp-amsterdam lib to v6.0.2 (#120788)

klaasnicolaas committed:
Bump gridnet lib to v5.0.1 (#120793)

piitaya committed:
Update frontend to 20240628.0 (#120785)

Co-authored-by: J. Nick Koston <nick@koston.org>

bdraco committed:
Fix missing f-string in loop util (#120800)

bdraco committed:
Fix blocking I/O in xmpp notify to read uploaded files (#120801)

detected by ruff in https://github.com/home-assistant/core/pull/120799

bdraco committed:
Fix unneeded dict values for MATCH_ALL recorder attrs exclude (#120804)

* Small cleanup to handling MATCH_ALL recorder attrs exclude

* Fix unneeded dict values for MATCH_ALL recorder attrs exclude

The exclude is a set so the dict values were not needed

* Fix unneeded dict values for MATCH_ALL recorder attrs exclude

The exclude is a set so the dict values were not needed

* Fix unneeded dict values for MATCH_ALL recorder attrs exclude

The exclude is a set so the dict values were not needed

cmroche committed:
Bump greeclimate to 1.4.6 (#120758)

joostlek committed:
Bump aiomealie to 0.5.0 (#120815)

joostlek committed:
Bump python-opensky to 1.0.1 (#120818)

allenporter committed:
Bump gcal_sync to 6.1.3 (#120278)

joostlek committed:
Add icons to Airgradient (#120820)

joostlek committed:
Use meal note as fallback in Mealie (#120828)

joostlek committed:
Add unique id to Mealie config entry (#120816)

frenck committed:
Bump version to 2024.7.0b5

bdraco committed:
Fix publish cancellation handling in MQTT (#120826)

thecode committed:
Bump aiowebostv to 0.4.1 (#120838)

tsvi committed:
Fix Jewish calendar unique id move to entity (#120842)

allenporter committed:
Rollback PyFlume to 0.6.5 (#120846)

mib1185 committed:
Bump pyfritzhome to 0.6.12 (#120861)

silamon committed:
Fix routes with transfer in nmbs integration (#120808)

joostlek committed:
Bump pizzapi to 0.0.6 (#120691)

mkmer committed:
Allow EM heat on from any mode in Honeywell (#120750)

EtienneSOU committed:
Fix Tado fan mode (#120809)

frenck committed:
Bump version to 2024.7.0b6

eifinger committed:
Bump here-routing to 1.0.1 (#120877)

Thomas55555 committed:
Bump aioautomower to 2024.6.4 (#120875)

yuxincs committed:
Add handling for different STATFLAG formats in APCUPSD (#120870)

* Add handling for different STATFLAG formats

* Just use removesuffix

dougiteixeira committed:
Add missing translations for device class in SQL (#120892)

dougiteixeira committed:
Add missing translations for device class in Template (#120893)

allenporter committed:
Bump ical to 8.1.1 (#120888)

jbouwh committed:
Bump incomfort-client dependency to 0.6.3 (#120913)

dougiteixeira committed:
Add missing translations for device class in Scrape (#120891)

sdb9696 committed:
Bump python-kasa to 0.7.0.2 (#120940)

marcelveldt committed:
Mark dry/fan-only climate modes as supported for Panasonic room air conditioner (#120939)

allenporter committed:
Bump gcal_sync to 6.1.4 (#120941)

balloob committed:
Bump openai to 1.35.1 (#120926)

Bump openai to 1.35.7

bdraco committed:
Downgrade logging previously reported asyncio block to debug (#120942)

edenhaus committed:
Improve add user error messages (#120909)

mj23000 committed:
Fix Bang & Olufsen jumping volume bar (#120946)

thecode committed:
Fix Shelly device shutdown (#120881)

frenck committed:
Bump version to 2024.7.0b7

joostlek committed:
Fix missing airgradient string (#120957)

rytilahti committed:
Minor polishing for tplink (#120868)

joostlek committed:
Bump airgradient to 0.6.1 (#120962)

starkillerOG committed:
Bump reolink-aio to 0.9.4 (#120964)

starkillerOG committed:
Do not hold core startup with reolink firmware check task (#120985)

joostlek committed:
Bump yt-dlp to 2024.07.01 (#120978)

starkillerOG committed:
Reolink replace automatic removal of devices by manual removal (#120981)

Co-authored-by: Robert Resch <robert@resch.dev>

jesserockz committed:
[ESPHome] Disable dashboard based update entities by default (#120907)

Co-authored-by: Paulus Schoutsen <balloob@gmail.com>

joostlek committed:
Remove Aladdin Connect integration (#120980)

emontnemery committed:
Create log files in an executor thread (#120912)

marcelveldt committed:
Handle mains power for Matter appliances (#121023)

marcelveldt committed:
Fix setting target temperature for single setpoint Matter thermostat (#121011)

frenck committed:
Bump version to 2024.7.0b8

bramkragten committed:
Update frontend to 20240702.0 (#121032)

jpbede committed:
Temporarily set apprise log level to debug in tests (#121029)

Co-authored-by: Franck Nijhof <git@frenck.dev>

frenck committed:
Bump version to 2024.7.0b9

bdraco committed:
Bump inkbird-ble to 0.5.7 (#121039)

changelog: https://github.com/Bluetooth-Devices/inkbird-ble/compare/v0.5.6...v0.5.7

frenck committed:
Bump version to 2024.7.0b10

eifinger committed:
Bump here-transit to 1.2.1 (#120900)

knyar committed:
Generate Prometheus metrics in an executor job (#121058)

bramkragten committed:
Update frontend to 20240703.0 (#121063)

Kane610 committed:
Bump axis to v62 (#121070)

marcelveldt committed:
Bump python-matter-server to 6.2.2 (#121072)

synesthesiam committed:
Bump intents to 2024.7.3 (#121076)

edenhaus committed:
Bump deebot-client to 8.1.0 (#121078)

marcelveldt committed:
Matter fix Energy sensor discovery schemas (#121080)

frenck committed:
Bump version to 2024.7.0b11

frenck committed:
Bump version to 2024.7.0
